### ISSUE TITLE
A "--skip-metadata" option is necessary

### ISSUE THREAD

**Issue by montagne27** [jbarlow83/OCRmyPDF#378]
Hello,

When OCRing PDFs downloaded from the Internet, sometimes the "ocrmypdf" command would fail, and the following exception would be raised:

```
    Exception #1
      'builtins.ValueError(Metadata seems to be XML but not XMP)' raised in ...
       Task = def ocrmypdf._weave.weave_layers(...):
```

If I first strip the PDFs from their metadata using "exiftool":

`exiftool -all:all= -o output.pdf input.pdf`

no exception is ever raised. This means that a "--skip-metadata" option would be very useful, and even necessary in some cases.

P.S. "ocrmypdf" already provides 4 metadata options ("--title", "--author", "--subject", "--keywords"), but they don't solve the problem.

**Comment by jbarlow83** [comment]
Can you point me to one of these files?

**Comment by montagne27** [comment]
I extracted a single page from one of the problematic PDFs, but it's still 28 MB, and Github won't let me upload it (10 MB limit). I put it here:

https://www.sendspace.com/file/w8ewle

Hope that's all right.

**Comment by jbarlow83** [comment]
The file contains an trivial XMP metadata packet. Fix will need to be in pikepdf.

```xml
<?xpacket begin="" id=""?>
<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="">
</x:xmpmeta>
<?xpacket end=""?>
```

**Comment by jbarlow83** [comment]
Fixed in v1.3.1... a while back.
